lua/lspconfig/server_configurations/julials.lua

```diff
@@ -1,5 +1,34 @@
 local util = require 'lspconfig.util'
 
+local function activate_env()
+  local bufnr = vim.api.nvim_get_current_buf()
+  local julials_client = util.get_active_client_by_name(bufnr, 'julials')
```

Comment:
use `vim.lsp.get_clients()`. This `util.get_active_client_by_name` function should be removed @glepnir

Reply:
In order to facilitate the transition of 0.9 users, which version will completely stop supporting 0.9x? 0.10.2 or 0.11.  then i can clean up utils including using vim.fs.

Reply:
Updated in latest commit revision

Reply:
> which version will completely stop supporting 0.9x?

We can drop 0.9 support now, if we want. But that's separate from the first step, which is to remove all possible usages of vim.util *in the configs in this repo* :)

Reply:
> It might be a good idea to keep the 0.9 support

Irrelevant for this PR. `vim.lsp.get_clients()` exists in 0.9

Reply:
Nope. it only in 0.10.

Reply:
> Irrelevant for this PR. `vim.lsp.get_clients()` exists in 0.9

I figured adding code that doesn't not match the current Neovim version support of the plugin would be irrelevant for this PR and a followup would make the blanket decision of which Neovim version to support. For now, I have removed the 0.9 support as requested

Reply:
If the idea is to remove all usages of `util` in the configuration. For now, should I include 0.9 support by checking for the old, deprecated function name?